Emby.Naming/Video/VideoListResolver.cs

```diff
@@ -152,22 +152,22 @@ public IEnumerable<VideoInfo> Resolve(List<FileSystemMetadata> files, bool suppo
                     .Where(i => i.ExtraType == ExtraType.Trailer)
                     .ToList();
 
-                list[0].Extras.AddRange(trailers);
+                trailers.AddRange(list[0].Extras);
+                list[0].Extras = trailers;
 
                 remainingFiles = remainingFiles
                     .Except(trailers)
                     .ToList();
             }
 
             // Whatever files are left, just add them
-            list.AddRange(remainingFiles.Select(i => new VideoInfo
+            list.AddRange(remainingFiles.Select(i => new VideoInfo(i.Name)
             {
                 Files = new List<VideoFileInfo> { i },
-                Name = i.Name,
                 Year = i.Year
             }));
 
-            return list.OrderBy(i => i.Name);
+            return list;
```

Comment:
again, ordering gone... why?

Reply:
Callers don’t depend on the order afaik and sorting is SLOW

Reply:
Who is calling that method then?

Reply:
`MovieResolver` and `LibraryManager`
`MovieResolver` does nothing with the result if it returns more than 1 item afaik
`LibraryManager` searches for an item with a specific path